shared/actions/login/index.js

```diff
@@ -429,7 +435,8 @@ function makeKex2IncomingMap (dispatch, getState) : incomingCallMapType {
               storeSecret: false
             }),
             onBack: () => dispatch(cancelLogin(response)),
-            error: retryLabel
+            error: retryLabel,
+            username
```

Comment:
This is kinda weird, shouldn't this be fetched from the store, and above be a dispatch instead of changing the username var?